docs/reference/commandline/trust_inspect.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "trust inspect"
+description: "The inspect command description and usage"
+keywords: "inspect, notary, trust"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# trust inspect
+
+```markdown
+Usage:  docker trust inspect [OPTIONS] IMAGE[:TAG]
+
+Display detailed information about keys and signatures
+
+```
+
+## Description
+
+Docker trust inspect provides detailed information on signed repositories.
```

Comment:
Mark this up: `docker trust inspect`. If you don't feel comfortable starting the sentence with lowercase mono text, then `The `docker trust inspect` command...."